ChangeLog

```diff
@@ -27,60 +34,43 @@
 - Added new icons for Linux desktop use (#1672).
   (contributed by @jujudusud)
 
-- Server: Improvements to multi-threading performance (#960)
-  (contributed by @menzels, @softins)
-
-- Github autobuild for Mac now uses Xcode 11.7 and SDK 10.15 for compatibility
-  with Qt5 (#1655).
-  (contributed by @softins)
-
 - GUI: Corrected handling of custom directory server in the server, to prevent
   unintended registration with a directory server (#1624, #1627).
   (contributed by @softins)
 
 - GUI: Corrected alignment of Mute icon above fader (#811, #1312, #1640).
   (contributed by @vimpostor)
 
-- The term "Central server" has been replaced with "Directory server" (#1407).
-  Note that the program will still accept the --centralserver option for backward
-  compatibility with existing system startup scripts.
-  (contributed by @pljones)
-
-- GUI: Settings window has been reorganized into tabs (#1415, #1554, #1542):
-    * User Profile window has been integrated into the settings window.
-    * Input Pan has been moved to the newly created Advanced tab and removed from main window.
-  (contributed by @dcorson-ticino-com, @pljones)
-
 - GUI: Support for more than two mixer rows has been added (#1549, #1560).
   (contributed by @pljones)
 
 - GUI: --clientname also works in dialog titles now (#1352, 1370).
   (contributed by @dcorson-ticino-com)
 
 - GUI: Translations have been updated
-  * Dutch, by @henkdegroot (#1562, #1623)
-  * French, by @jujudusud (#1648)
+  * Dutch, by @henkdegroot (#1562, #1623, #1714, #1557)
+  * French, by @jujudusud (#1648, #1708)
   * German, by @rolamos (#1677)
   * Italian, by @dzpex (#1620)
   * Polish, by @SeeLook (#1619)
   * Portuguese Brazilian, by @melcon (#1671)
   * Portuguese European, by @Snayler (#1689)
   * Slovak, by @jose1711 (#1647)
-  * Spanish, by @ignotus666 (#1621)
+  * Spanish, by @ignotus666 (#1621, #1730)
   * Swedish, by @genesisproject2020 (#1664, #1696)
 
-- Client: Automatic channel fader adjustment simplifies mixer setup by using the channel level meters (#1071).
-  (contributed by @JohannesBrx)
-
-- Client: Basic audio feedback detection has been added (#1179).
-  (contributed by @JohannesBrx)
-
 - Network: Support for DSCP Quality of Service flags has been added (#1310).
   This is supposed to lead to improved network performance.
   It is enabled by default.
   On Windows, this requires additional configuration in order to work.
```

Comment:
```suggestion
  On Windows, this requires additional configuration in order to work. Please see [the Tips & Tricks page on the website](https://jamulus.io/wiki/Tips-Tricks-More#quality-of-service) for a setup guide for Windows.
```

Reply:
Adding that link is a slightly unsafe instruction perhaps as this will be moved into the Server Administrators Manual later.

Reply:
I have now added what @ann0see suggested except for the direct link due to @gilgongo's concerns.